src/azure-cli/azure/cli/command_modules/acr/_archive_utils.py

```diff
@@ -69,7 +69,9 @@ def upload_source_code(cmd, client,
 def _pack_source_code(source_location, tar_file_path, docker_file_path, docker_file_in_tar):
     logger.warning("Packing source code into tar to upload...")
 
-    ignore_list, ignore_list_size = _load_dockerignore_file(source_location)
+    # NOTE: os.path.basename is unable to parse "\" in the file path
+    original_docker_file_name = os.path.basename(docker_file_path.replace("\\", "/"))
+    ignore_list, ignore_list_size = _load_dockerignore_file(source_location, original_docker_file_name)
```

Comment:
where is `ignore_list_size` used? do you think its redundant to return that?

Reply:
Hi @shahzzzam, thanks for your comment. It looks like it's used in:

https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acr/_archive_utils.py#L107

I am not sure if it would be appropriate to refactor that, but I personally see that as out of scope for this PR. Please let me know if you have suggestions in that sense.